graphbolt/include/graphbolt/csc_sampling_graph.h

```diff
@@ -161,25 +65,14 @@ class CSCSamplingGraph : public torch::CustomClassHolder {
   /** @brief Get the index tensor. */
   const torch::Tensor Indices() const { return indices_; }
 
-  /** @brief Check if the graph is heterogeneous. */
-  inline bool IsHeterogeneous() const { return hetero_info_ != nullptr; }
-
   /** @brief Get the node type offset tensor for a heterogeneous graph. */
```

Comment:
Explain more about what's node type offset tensor, e.g. we make the id of node of different node types continuous, the offset is blablabla.

Reply:
added